### PR TITLE
Add process cpu normalized

### PR DESCRIPTION
**Pull request by trask**
~currently built on top of #2176, will rebase once that is merged~

~One outstanding question about whether we should apply the config option to system cpu percentage also.~

requires doc update

### CODE REVIEW

File: agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulse.java [hunk 1]
```diff
@@ -77,7 +83,8 @@ private void initialize(
       Supplier<URL> endpointUrl,
       Supplier<String> instrumentationKey,
       @Nullable String roleName,
-      @Nullable String roleInstance) {
+      @Nullable String roleInstance,
+      boolean backCompatNonNormalizedCpuPercentage) {
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulse.java:87]: this is confusing.. backCompatNonNormalizedCpuPercentage is used normalized cpu percentage.. 
can you rename to usedNormalizedCpuPercent? and add the explaination for backward compatibility of non-normalized cpu percentage?

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java [hunk 2]
```diff
@@ -257,6 +257,11 @@ public String toString() {
     // telemetry (they are much smaller so a larger queue size is ok)
     public int metricsExportQueueCapacity = 65536;
 
+    // unfortunately the Java SDK behavior has always been to report the "% Processor Time" number
+    // as "normalized" (divided by # of CPU cores), even though it should be non-normalized
+    // maybe this can be fixed in 4.0 (would be a breaking change)
+    public boolean backCompatNonNormalizedCpuPercentage;
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java:263]: usedNormalizedCpuPercentage?

Comment (trask) [reply agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java:267]: ya, this is confusing, I added more comments, I'm not sure what to do about the name, we already report a normalized cpu percentage so I'm not sure `useNormalizedCpuPercentage` makes it any clearer.

maybe `useNormalizedValueForNonNormalizedCpuPercentage` (with default `true` representing the backwards compatible value)? kinda long but... 😂

Comment (heyams) [reply agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java:267]: useNormalizedValueForNonNormalizedCpuPercentage sounds good although it's a bit long but it's clearer.

**Review by heyams — APPROVED** [review]

**Pull request merged**